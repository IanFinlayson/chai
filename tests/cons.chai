def main():
    var ls = 1 :: 2 :: 3 :: 4 :: [5, 6]
    print(ls)
    
    var i = 10
    var squares = []
    while i >= 0:
        squares = i ** 2 :: squares
        i = i - 1
    print(squares)

