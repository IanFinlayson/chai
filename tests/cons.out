[1, 2, 3, 4, 5, 6]
[0, 1, 4, 9, 16, 25, 36, 49, 64, 81, 100]
